Packages/KeyAppKit/Sources/BankTransfer/Service/StrigaBankTransferService.swift

```diff
@@ -1,14 +1,34 @@
 import Combine
 import Foundation
 
-public final class StrigaBankTransferService: BankTransferService {
-
-    public var userData: AnyPublisher<UserData, Never> { subject.eraseToAnyPublisher() }
+private extension String {
+    static let expectedIncomingTxVolumeYearly = "MORE_THAN_15000_EUR"
```

Comment:
seems like it should be enum since it's a registration data option or if it's fixed value just make it private static not global String